[2021-07-09 18:58:27,115] [loging.py|__console] [line:40] INFO    : *****正在执行-----正常登录-----用例*****
[2021-07-09 18:58:27,117] [loging.py|__console] [line:40] INFO    : *****开始发送请求*****
[2021-07-09 18:58:27,318] [loging.py|__console] [line:40] INFO    : *****发送请求结束*****
[2021-07-09 18:58:27,322] [loging.py|__console] [line:46] ERROR   : {'code': 0, 'message': '成功', 'data': {'fddTemplateId': 'XYFDD6776032657531236352', 'isUpdatePwd': 0, 'loginType': 0, 'token': 'eyJhbGciOiJIUzUxMiJ9.eyJkYXRhIjp7ImZkZFRlbXBsYXRlSWQiOiJYWUZERDY3NzYwMzI2NTc1MzEyMzYzNTIiLCJpc1VwZGF0ZVB3ZCI6MCwibG9naW5UeXBlIjowLCJ1c2VyTG9naW5JZCI6NjU5Mzc0NjkwMTM1MzA4Mjg4MCwibWVyY2hhbnRJZCI6IiIsInRhc2tVcmwiOiJodHRwczovL2Nkbi5sYXN0bWlsZXMuY24vYXBwcy82NzIyODEyMjM3NzQzNTk5NjE2LnBkZiIsInN5c05hbWUiOiLns7vnu5_mk43kvZzmiYvlhowtVjEuMCIsImlzTWFuYWdlciI6MSwiaXNBc3luY1JlcXVlc3QiOiJ0cnVlIiwic3lzVXJsIjoiaHR0cHM6Ly9jZG4ubGFzdG1pbGVzLmNuL2FwcHMvNjcyMjgxMTU2OTM5ODAzNDQzMi5wZGYiLCJ0YXNrTmFtZSI6IuWFheWAvOS7o-WPkeaTjeS9nOaJi-WGjC1WMS4wIiwibWVyY2hhbnRSYW5nZSI6MSwidXNlcm5hbWUiOiJhZG1pbiJ9LCJzdWIiOiI2NTkzNzQ2OTAxMzUzMDgyODgwIiwiZXhwIjoxNjI3OTc1NzkyfQ.0hmDhEuINml_EBc7XaLO-gMWA15H9CAzDLDGaIy67uhh1VQqEyknXSQ9JZf7qhLkHVNc5FeErbUYFqbTHFDGPw', 'userLoginId': '6593746901353082880', 'merchantId': '', 'taskUrl': 'https://cdn.lastmiles.cn/apps/6722812237743599616.pdf', 'sysName': '系统操作手册-V1.0', 'sysLimit': 20000.0, 'isManager': 1, 'isAsyncRequest': 'true', 'sysUrl': 'https://cdn.lastmiles.cn/apps/6722811569398034432.pdf', 'taskName': '充值代发操作手册-V1.0', 'merchantRange': 1, 'username': 'admin'}, 'requestId': 'ac130005162582830892459516'}
[2021-07-09 18:58:27,329] [loging.py|__console] [line:40] INFO    : *****正在执行-----账号错误登录-----用例*****
[2021-07-09 18:58:27,331] [loging.py|__console] [line:40] INFO    : *****开始发送请求*****
[2021-07-09 18:58:27,426] [loging.py|__console] [line:40] INFO    : *****发送请求结束*****
[2021-07-09 18:58:27,431] [loging.py|__console] [line:46] ERROR   : {'code': 500, 'message': '账号不存在！', 'data': None, 'requestId': 'ac130005162582830910459526'}
[2021-07-09 18:58:27,436] [loging.py|__console] [line:40] INFO    : *****正在执行-----密码错误登录-----用例*****
[2021-07-09 18:58:27,438] [loging.py|__console] [line:40] INFO    : *****开始发送请求*****
[2021-07-09 18:58:27,549] [loging.py|__console] [line:40] INFO    : *****发送请求结束*****
[2021-07-09 18:58:27,550] [loging.py|__console] [line:46] ERROR   : {'code': 500, 'message': '密码错误！您还有4次尝试机会', 'data': None, 'requestId': 'ac130005162582830920559536'}
[2021-07-09 18:58:27,560] [loging.py|__console] [line:40] INFO    : *****正在执行-----验证码错误登录-----用例*****
[2021-07-09 18:58:27,561] [loging.py|__console] [line:40] INFO    : *****开始发送请求*****
[2021-07-09 18:58:27,637] [loging.py|__console] [line:40] INFO    : *****发送请求结束*****
[2021-07-09 18:58:27,643] [loging.py|__console] [line:46] ERROR   : {'code': 500, 'message': '验证码错误或已过期', 'data': None, 'requestId': 'ac130005162582830933059546'}
